src/app/window-actions.ts

```diff
@@ -20,6 +20,7 @@ enum Permissions {
     OPEN_EXTERNAL = 'openExternal',
 }
 const PERMISSIONS_NAMESPACE = 'Permissions';
+const { isCustomTitleBar } = config.getUserConfigFields([ 'isCustomTitleBar' ]);
```

Comment:
`isCustomTitleBar` is a public instance and can be accessed directly from `windowHandler.isCustomTitleBar`

Reply:
Noted, will change 👍